test/unit/typedQueryBuilderTests.ts

```diff
@@ -1337,17 +1337,72 @@ describe('TypedKnexQueryBuilder', () => {
 
     it('should handle two levels of where exists', (done) => {
         const typedKnex = new TypedKnex(knex({ client: 'postgresql' }));
-        const query = typedKnex.query(User).whereExists(User, (subQuery => {
+        const query = typedKnex.query(User).whereExists(User, (subQuery) => {
             subQuery.whereColumn('status', '=', 'status').whereColumn('id', '<>', 'id');
 
             subQuery.whereExists(User, (subQuery2 => {
                 subQuery2.whereColumn('status', '=', 'status').whereColumn('id', '<>', 'id');
             }));
-        }));
+        });
 
         const queryString = query.toQuery();
         assert.equal(queryString, `select * from "users" where exists (select * from "users" as "subquery0$users" where "subquery0$users"."weirdDatabaseName" = "users"."weirdDatabaseName" and "subquery0$users"."id" <> "users"."id" and exists (select * from "users" as "subquery0$subquery0$users" where "subquery0$subquery0$users"."weirdDatabaseName" = "subquery0$users"."weirdDatabaseName" and "subquery0$subquery0$users"."id" <> "subquery0$users"."id"))`);
 
         done();
     });
+
+    it('should get correct column for nested queries', (done) => {
+        const typedKnex = new TypedKnex(knex({ client: 'postgresql' }));
+        let nestedStatusColumnName;
+        // let nestedUserCategoryYear;
```

Comment:
I tried fiddling around but I wasn't able to get this to work.

If you pass 1 argument to `getColumnName`, you get back the table name and the property. For instance, `'id'` becomes `'users.id'`. If you pass multiple arguments to `getColumnName`, you only get the property and its properties, for example: `'category', 'year'` becomes `'category.year'` instead of the desired `'user.category.year'`. That means nested properties aren't able to be gotten easily with `getColumnNameWithQuotes`, and thus nested subqueries won't work if you're looking for nested properties.

If it's a desired feature, maybe we can try getting the table name back from `getColumnName`. If not then we can remove the comments in the code

Reply:
Can you try again with `getColumnAlias`?

Reply:
I commented on it in the current version